contrib/packs/actions/workflows/install.yaml

```diff
@@ -98,6 +99,7 @@ tasks:
       packs: <% ctx().packs_list %>
       env: <% ctx().env %>
       python3: <% ctx().python3 %>
```

Comment:
i'm wondering if this timeout should also be applied to other sub-actions such as the pack content registration (thinking EC2 pack where there is 1000+ actions)